src/test/java/uk/gov/hmcts/reform/professionalapi/repository/PrdEnumRepositoryTest.java

```diff
@@ -20,7 +20,7 @@
     public void test_findAll() {
         List<PrdEnum> prdEnums = prdEnumRepository.findAll();
 
-        assertThat(prdEnums).hasSize(37);
+        assertThat(prdEnums).hasSize(39);
         assertThat(prdEnums.get(0).getPrdEnumId().getEnumCode()).isEqualTo(0);
     }
 
```

Comment:
should nt u have a method here to check for new roles?

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/557/commits/28f8adb4ab9d1d0436f097d64771150a9f365e36